### PR TITLE
*: support sourceID on operate-source stop

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
part of #826

### What is changed and how it works?
support sourceID on operate-source stop

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: dm/ctl/master/operate_source.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 // NewOperateSourceCmd creates a OperateSource command
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> [config-file ...] [--print-sample-config]",
+		Use:   "operate-source <operate-type> [config-file | sourceID ...] [--print-sample-config]",
```
Comment (csuzhangxc) [inline dm/ctl/master/operate_source.go:34]: How about updating the message of `operate-source create/update/stop should specify config-file(s)`?

Comment (lance6716) [reply dm/ctl/master/operate_source.go:34]: what's the correct way to express 😂 
```
operate-source create|update [config-file ...] [--print-sample-config]
operate-source stop [config-file | sourceID ...] [--print-sample-config]
operate-source show [--print-sample-config]
```
is this OK?

Comment (csuzhangxc) [reply dm/ctl/master/operate_source.go:34]: OK. 😂

Comment (lance6716) [reply dm/ctl/master/operate_source.go]: https://github.com/pingcap/dm/pull/855/commits/3780ae509e1da536483d49d6f6dd9f6c49801cc5

Comment (lance6716) [reply dm/ctl/master/operate_source.go]: multi-line usage is quite hard using cobra

```
[2020-08-04T04:16:35.427Z] Available Commands:
[2020-08-04T04:16:35.427Z]   check-task          	check-task <config-file>
[2020-08-04T04:16:35.427Z]   get-task-config     	get-task-config <task-name> [--file filename]
[2020-08-04T04:16:35.427Z]   list-member         	list-member [--leader] [--master] [--worker] [--name master-name/worker-name ...]
[2020-08-04T04:16:35.427Z]   migrate-relay       	migrate-relay <source> <binlogName> <binlogPos>
[2020-08-04T04:16:35.427Z]   offline-member      	offline-member <--master/--worker> <--name master-name/worker-name>
[2020-08-04T04:16:35.427Z]   operate-leader      	operate-leader <operate-type>
[2020-08-04T04:16:35.427Z]   operate-schema      	operate-schema <operate-type> <-s source ...> <task-name> <-d database> <-t table> [schema-file]
[2020-08-04T04:16:35.427Z]   operate-source      	operate-source create|update [config-file ...] [--print-sample-config] [flags]
[2020-08-04T04:16:35.427Z]         operate-source stop [config-file | sourceID ...] [--print-sample-config] [flags]
[2020-08-04T04:16:35.427Z]         operate-source show [--print-sample-config] [flags]
```

Comment (lance6716) [reply dm/ctl/master/operate_source.go]: how about just hide support `sourceID`, and find a better way to display when we refactor subcommand of https://github.com/pingcap/dm/issues/398

Comment (csuzhangxc) [reply dm/ctl/master/operate_source.go]: > how about just hide support sourceID, and find a better way to display when we refactor subcommand of #398

OK

File: dm/ctl/master/operate_source.go [hunk 2]
```diff
@@ -31,7 +31,11 @@ import (
 // NewOperateSourceCmd creates a OperateSource command
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> [config-file ...] [--print-sample-config]",
+		// the leading space could provide more friendly terminal printing
+		// cobra.Command doesn't support multi-line usage well
+		Use: `operate-source create|update [config-file ...] [--print-sample-config] [flags]
+        operate-source stop [config-file | sourceID ...] [--print-sample-config] [flags]
```
Comment (GMHDBJD) [inline dm/ctl/master/operate_source.go:37]: --print-sample-config cannot work with create/update/stop at the same time. But change to print-sample-config look strange. 😖

Comment (lance6716) [reply dm/ctl/master/operate_source.go:37]: seems `--print-sample-config` will overwrite the result of create/update/stop/show, I guess user will only use `--print-sample-config` when they don't know how to write a config file (thus they can't do a right source operation), so it's not confusing.

going to change usage to
```
operate-source <operate-type> [config-file | sourceID ...] [--print-sample-config]
```

File: dm/ctl/master/operate_source.go (hunk 1 above)
Comment (lance6716) [reply dm/ctl/master/operate_source.go]: revert in https://github.com/pingcap/dm/pull/855/commits/b41bf2c225e91b2bb5f10ef8d34dce0bc4950b8e

**Pull request merged**